### PR TITLE
Fix the display of the phone numbers info message

### PR DESCRIPTION
**Pull request by SLiCK-303**
When you switched the Phone number is mandatory's value to NO, the info message You must register at least one phone number. is still appear, and this is wrong.

### CODE REVIEW

**Comment by SLiCK-303** [comment]
How to test: BO > Preferences > Customer > (Phone number is mandatory = NO), FO > My addresses > Add a new address, check if the info message "You must register at least one phone number." is disappeared.

File: address.tpl [hunk 1]
```diff
@@ -166,8 +181,8 @@
         <input class="validate form-control" data-validate="{$address_validation.phone_mobile.validate}" type="tel" id="phone_mobile" name="phone_mobile" value="{if isset($smarty.post.phone_mobile)}{$smarty.post.phone_mobile}{else}{if isset($address->phone_mobile)}{$address->phone_mobile|escape:'html':'UTF-8'}{/if}{/if}" />
       </div>
     {/if}
-    {if isset($one_phone_at_least) && $one_phone_at_least && !$atLeastOneExists}
-      <p class="help-block required">{l s='You must register at least one phone number.'}</p>
+    {if isset($one_phone_at_least) && $one_phone_at_least && !$atLeastOneExists && !$phoneIsRequired}
```
Comment (firstred) [inline address.tpl:184]: Shouldn't this be `$phoneIsRequired`?

Comment (SLiCK-303) [reply address.tpl:184]: seems to work this way..

Comment (firstred) [reply address.tpl:184]: I don't believe that :P 
Let me double check first.

**Comment by firstred** [comment]
I am going to try to find an alternative in the coming days. This PR does not seem to work on my test environment.

If you know what could be done, don't hesitate to share. I'm confused atm :S

**Comment by Dh42** [comment]
Thanks for the fix, I had to manually pick it. https://github.com/thirtybees/community-theme-default/commit/91c22b6ee60e46520a6aa390e347400eab721a00

**Pull request closed without merging**